### ISSUE TITLE
Can't setup 2FA - won't accept backup code

### ISSUE THREAD

**Issue by jbirnick** [mozilla/fxa#17853]
## Description

I just created a Mozilla Account to publish my add-on. For publishing my add-on, I need to setup 2FA. However, after confirming a 2FA code, it would ask me to confirm with a backup code. And then no matter what backup code from the previous page I give them, it would always just say "there was a problem confirming your backup authentication code".

I found another user on reddit who had the same problem a month ago: https://www.reddit.com/r/firefox/comments/1f8g4pb/cant_add_2fa_to_mozilla_account/

![image](https://github.com/user-attachments/assets/9ae44636-818a-4bf6-bf8a-b08bb520682b)

### Steps to reproduce

Create a new Mozilla account and try to setup 2FA.

### Environment

Arch Linux. Tried with both Firefox and Chromium.

**Comment by ReneDussault** [comment]
i have the exact same issue. copy pasted from the codes, but at step 3, nothing works. had to wait 15 minutes, then tried again with authy, code gets generated, enter the authy code, everything is fine.. then we get to the backup code, copy directly from the window that asks you to download and backup this file.. then add one of the code into step 3, fail

**Comment by jbirnick** [comment]
Yeah same.

**Comment by ReneDussault** [comment]
> Yeah same.

it JUST allowed me to make it happen. im not sure why, but now it works... idk, weird bug

**Comment by clouserw** [comment]
99% of the time issues like these are that a clock on one of the devices is off.  Even a few seconds can matter.  Would you verify that your devices clocks are accurate and sync'd to a timeserver?

**Comment by jbirnick** [comment]
I don't think it was related to that; it also worked for me when it worked for ReneDussault.

Also, for the backup codes it should not be related to any timing, right? Again, providing the 6-digit authentication code _did_ work, but providing the backup code (which is _not_ timing-based) did not work.

**Comment by clouserw** [comment]
Right.  Timing only matters for the 2FA codes

**Comment by jbirnick** [comment]
Yeah that's the same thing.

**Comment by gustavoeso** [comment]
I got the same thing, try to setup, won't accept recovery codes for some reason

**Comment by prefabcode** [comment]
Got the same issue. Trying to create a developer account so I can submit an addon, but I cannot get past setting up this 2FA stuff. I set up the authenticator, copy codes to a safe place, then in step 3 it asks me to confirm one of the codes. I copy one of the codes and get this "There was a problem confirming your backup authentication code".

**Comment by jbirnick** [comment]
Well but I did not sign up with Google/Apple, so I had a password, and it still didn't work.

**Comment by vbudhram** [comment]
Hello 👋🏽, a fix for this has landed in https://github.com/mozilla/fxa/pull/17893. Thanks again for the report!

**Comment by Noxor11** [comment]
I see this fix is already merged and I'm not sure if the changes are up already, but I still had the same error several times just now. I'm still not able to confirm the backup authentication code.

**Comment by vbudhram** [comment]
Hey Noxor, I'll see if this can be uplifted but otherwise we expect the next production release to be Nov 13.
